website/docs/docs/dbt-cloud/setting-up-dbt-cloud/configure-a-repository/git-url.md

```diff
@@ -15,25 +16,38 @@ After importing a project by Git URL, dbt Cloud will generate a Deploy Key for y
 
 :::info Use GitHub?
 
-If you use GitHub, you can import your repo directly using [dbt Cloud's GitHub Application](cloud-installing-the-github-application). Connecting your repo via the GitHub Application [enables Continuous Integration](cloud-enabling-continuous-integration-with-github) for scheduled dbt runs.
+If you use GitHub, you can import your repo directly using [dbt Cloud's GitHub Application](github-repository). Connecting your repo via the GitHub Application [enables Continuous Integration](cloud-enabling-continuous-integration-with-github) for scheduled dbt runs.
 
 :::
 
-To add a deploy key to a GitHub account, navigate to the Deploy keys tab of the settings page in your GitHub repository. After supplying a name for the deploy key and pasting in your deploy key (generated by dbt Cloud), be sure to check the "Allow write access" checkbox. After adding this key, dbt Cloud will be able to read and write files in your dbt project.
+To add a deploy key to a GitHub account:
+1. Navigate to the Deploy keys tab of the settings page in your GitHub repository.
+2. Supply a name for the deploy key and paste in your deploy key (generated by dbt Cloud)
+3. Be sure to check the "Allow write access" checkbox.
+
+dbt Cloud will be able to read and write files in your dbt project.
 
 See also:  [Adding a deploy key in GitHub](https://github.blog/2015-06-16-read-only-deploy-keys/)
 
 <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/cd7351c-Screen_Shot_2019-10-16_at_1.09.41_PM.png" title="Configuring a GitHub Deploy Key"/>
 
 ### BitBucket
 
-To add a deploy key to a BitBucket account, navigate to "SSH keys" tab in the Personal Settings page of your BitBucket account. Next, click the "Add key" button and paste in the deploy key generated by dbt Cloud for your repository. After saving this SSH key, dbt Cloud will be able to read and write files in your BitBucket repository.
+To add a deploy key to a BitBucket account
+1. Navigate to "SSH keys" tab in the Personal Settings page of your BitBucket account.
+2. Click the "Add key" button and paste in the deploy key generated by dbt Cloud for your repository.
+
+After saving this SSH key, dbt Cloud will be able to read and write files in your BitBucket repository.
```

Comment:
do you think we should add a "NOTE: this needs to be configured on a personal setting, NOT the repo" since we get so many questions about it?